src/plugins/data/public/ui/filter_bar/filter_bar.tsx

```diff
@@ -112,7 +112,7 @@ function FilterBarUI(props: Props) {
           repositionOnScroll
         >
           <EuiFlexItem grow={false}>
-            <div style={{ width: 400 }}>
+            <div style={{ width: 420, maxWidth: '100%' }}>
```

Comment:
`400` -> `420` this affects only flow of creation new filter. I noticed that width when editing existing filter is 420. So I also changed it here to make the width during creation and editing the same